.github/workflows/auto-approver.yaml

```diff
@@ -5,12 +5,13 @@ on:
   push:
     branches:
       - "renovate/**"
+      - "automated-update-charm-libs"
 
 jobs:
   autoapprove:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
are these intended to be committed?

Reply:
I thought we were updating to checkout v4. Here's the current auto-approver.yaml in hydra: https://github.com/canonical/hydra-operator/blob/main/.github/workflows/auto-approver.yaml